src/components/Calculator.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
```

Comment:
- [ ] Kindly create at least one child component for the `Calculator` component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component).

Reply:
I have done the aforementioned changes as you suggested.